### ISSUE TITLE
Update Common WordPress errors with block theme info

### ISSUE THREAD

**Issue by annezazu** [WordPress/Documentation-Issue-Tracker#104]
Need to update this page: https://wordpress.org/support/article/common-wordpress-errors/ in light of [this slack convo](https://wordpress.slack.com/archives/C02RP4VMP/p1637080950219600?thread_ts=1636999727.203900&cid=C02RP4VMP). Essentially, need to include information about troubleshooting the age of block themes:

> There are two ways to essentially create a “Blank Slate” to troubleshoot against:
> - Activate Twenty Twenty-Two, which will let you test against the FSE functionality built into 5.9+
> - Activate Twenty Twenty-One, which will turn off the site editor and re-enable the Customizer by default.

**Comment by carolinan** [comment]
Should this page include something about common errors when you use a block theme? Like the message for when templates are not found?

**Comment by carolinan** [comment]
(Or do we wait to see if this is a problem that users start reporting and asking about)

**Comment by annezazu** [comment]
Hmm good question. Right now, I lean towards _not yet_ and instead updating as errors come in. The confidence in my answer is very low here though and I'd likely need help documenting the various things folks would want to see.

**Comment by annezazu** [comment]
Added in this language throughout the doc and scheduled the update for Jan 25th, 2022 with the others:

> If you are using WordPress 5.8 and below, please switch to Twenty Twenty-One since the Twenty Twenty-Two theme requires 5.9 and above. 

I also changed phrasing to "the default theme" to "a default theme" to indicate that folks shouldn't just switch to the latest. This is an extremely slight update but I am hoping to be iterative here since I imagine it's an often used page.

cc @zzap for further thoughts too! For now, going to close this out but happy to reopen.
